.github/workflows/jira-creation.yml

```diff
@@ -13,13 +13,13 @@ name: Jira Issue Creation
 on:
   issues:
     types: [opened, labeled]
-    
+
 permissions:
   issues: write
 
 jobs:
   call-label-action:
-    uses: dbt-labs/jira-actions/.github/workflows/jira-creation.yml@main
+    uses: dbt-labs/actions/.github/workflows/jira-creation.yml@main
```

Comment:
Similar comment here, convert to [jira-creation-actions.yml](https://github.com/dbt-labs/actions/blob/main/.github/workflows/jira-creation-actions.yml)